packages/eslint-config/index.js

```diff
@@ -0,0 +1,128 @@
+module.exports = {
```

Comment:
Not super relevant to this PR, but I tried to create a package for our eslint config, that we can share among our packages. We haven't really discussed this, so I am okay with removing this again, but thought I'd at least share the idea @adrien2p @kasperkristensen